cluster-assign-roles.sh

```diff
@@ -47,7 +47,7 @@ function install_machines {
     else
       printf "About to bootstrap node $fqdn in $ENVIRONMENT run_list ${run_list}...\n"
       ./chefit.sh $ip $ENVIRONMENT
-      sudo -E knife bootstrap -E $ENVIRONMENT -r "$run_list" $ip -x ubuntu ${passwd:+-P} $passwd $KNIFE_ADMIN --sudo <<< $passwd
+      sudo -E knife bootstrap --node-ssl-verify-mode none --no-node-verify-api-cert -E $ENVIRONMENT -r "$run_list" $ip -x ubuntu ${passwd:+-P} $passwd $KNIFE_ADMIN --sudo <<< $passwd
```

Comment:
I would really prefer we verify the cert here, even though it's usually self-signed. Maybe we could add some `knife ssl fetch` command at some point before this gets run? Of course that can be tricky, as it will require some way of verifying the signature of the fetched cert, which you'll only have after it gets generated. Either way, it's probably out of scope for this PR :(